src/Templates/MLDotNet/Microsoft.VisualStudio.Templates.MLDotNet/Content/ConsoleApp/Program.cs

```diff
@@ -20,34 +18,35 @@ static void Main()
             var mlContext = new MLContext();
 
             // 1a. Load the training data using a TextLoader.
-            var reader = mlContext.Data.TextReader(
-                new[]
+            var reader = mlContext.Data.CreateTextReader(
```

Comment:
You can use the new convenience API to create the TextReader from the `SentimentData` type automatically.

```C#
            var reader = mlContext.Data.CreateTextReader<SentimentData>(hasHeader: true);

... // then later

        public class SentimentData
        {
            [LoadColumn(0)]
            public bool Label { get; set; }
            [LoadColumn(1)]
            public string Text { get; set; }
        }
```